.github/workflows/create-azure-machines.ps1

```diff
@@ -65,7 +68,11 @@ if ($SubscriptionId) {
     Connect-AzAccount -SubscriptionId $subscriptionId
 }
 
-Write-Host "Creating Azure Resources ($ResourceGroupName, $Location, $os, $VMSize)"
+if ($VMAlreadyExists) {
```

Comment:
The first part of `Add-NetPerfVm` already checks and skips if it's there.

Reply:
You're right, this isn't needed.